WMFComponents/Sources/WMFComponents/Components/Year in Review/WMFSlideShow.swift

```diff
@@ -64,4 +84,5 @@ public protocol SlideShowProtocol {
     var imageName: String { get }
     var imageOverlay: String? { get }
     var textOverlay: String? { get }
+    var infoURL: String? { get }
```

Comment:
This is minor but your `infoURL` name makes me think this type should be `URL?` instead of a `String` type. I think it would work fine if you changed this type to `URL?`.